packages/search/tests/integration/components/navi-search-result/asset-test.js

```diff
@@ -0,0 +1,44 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, findAll } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import Service from '@ember/service';
+import { set } from '@ember/object';
+
+module('Integration | Component | navi-search-result-asset', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  let service;
+
+  hooks.beforeEach(async function() {
+    // Load metadata needed for request fragment
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    service = this.owner.lookup('service:navi-search-provider');
```

Comment:
Can we just mock the data that the component expects to keep this tests independent of the service?
If you do want to test both, maybe it should be an acceptance test.